docs/metrics/getting-started-prometheus-grafana/README.md

```diff
@@ -23,46 +23,73 @@ cd getting-started-prometheus
 dotnet run
 ```
 
-Add a reference to [Prometheus
-Exporter Http Listener](../../../src/OpenTelemetry.Exporter.Prometheus.HttpListener/README.md):
+Add reference to [Console
+Exporter](../../../src/OpenTelemetry.Exporter.Console/README.md) and [OTLP
+Exporter](../../../src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md):
 
 ```sh
-dotnet add package OpenTelemetry.Exporter.Prometheus.HttpListener --prerelease
+dotnet add package OpenTelemetry.Exporter.Console
+dotnet add package OpenTelemetry.Exporter.OpenTelemetryProtocol
 ```
 
-Now, we are going to make some small tweaks to the example in the
-getting-started metrics `Program.cs` to make the metrics available via
-OpenTelemetry Prometheus Exporter.
+Now copy the code from [Program.cs](./Program.cs).
 
-First, copy and paste everything from getting-started metrics
-[example](../getting-started-console/Program.cs) to the Program.cs file of the
-new console application (getting-started-prometheus) we've created.
+### Check results in the console
 
-And replace the below line:
+Run the application again and we should see the metrics output from the console:
 
-```csharp
-.AddConsoleExporter()
+```text
+> dotnet run
+Press any key to exit
+
+Resource associated with Metric:
+    telemetry.sdk.name: opentelemetry
+    telemetry.sdk.language: dotnet
+    telemetry.sdk.version: 1.6.1-alpha.0.23
+    service.name: unknown_service:getting-started-prometheus-grafana
+
+Export MyFruitCounter, Meter: MyCompany.MyProduct.MyLibrary/1.0
+(2023-09-22T20:40:22.2586791Z, 2023-09-22T20:40:31.1582923Z] color: red name: apple LongSum
+Value: 54
+(2023-09-22T20:40:22.2586791Z, 2023-09-22T20:40:31.1582923Z] color: yellow name: lemon LongSum
+Value: 63
+(2023-09-22T20:40:22.2586791Z, 2023-09-22T20:40:31.1582923Z] color: green name: apple LongSum
+Value: 18
+
+...
 ```
 
-with
+Note that we have configured two exporters in the code:
 
 ```csharp
-.AddPrometheusHttpListener()
+using var meterProvider = Sdk.CreateMeterProviderBuilder()
+    ...
+    .AddConsoleExporter()
+    .AddOtlpExporter(options =>
+    {
+        options.Endpoint = new Uri("http://localhost:9090/api/v1/otlp/v1/metrics");
+        options.Protocol = OpenTelemetry.Exporter.OtlpExportProtocol.HttpProtobuf;
+    })
+    .Build();
 ```
 
-`PrometheusHttpListener` is a wrapper that contains `PrometheusExporter`. With
-`AddPrometheusHttpListener()`, OpenTelemetry `PrometheusExporter` will export
-data via the endpoint defined by
-[PrometheusHttpListenerOptions.UriPrefixes](../../../src/OpenTelemetry.Exporter.Prometheus.HttpListener/README.md#uriprefixes),
-which is `http://localhost:9464/` by default.
+When we run the application, the `ConsoleExporter` was printing the metrics on
```

Comment:
minor: inconsistent verb tense

Reply:
Good catch, fixed (also fixed the Jaeger doc).